docs/recipes/install/common/stateful.tex

```diff
@@ -2,42 +2,40 @@
 memory in a {\em stateless} configuration. Alternatively, Warewulf can also be
 used to partition and format persistent storage such that the VNFS image can be
 installed locally to disk in a {\em stateful} manner.  This does, however,
-require that a boot loader (GRUB) be added to the image as follows:
+require that a boot loader (GRUB) be added to the image. Enabling stateful nodes
+also requires additional site-specific, disk-related parameters in the Warewulf
+configuration. In the example that follows, the compute node configuration is
+updated to define where to install the GRUB bootloader, which disk to partition,
+which partition to format, and what the filesystem layout will look like.
 
+% begin_ohpc_run
+% ohpc_command if [[ ${enable_stateful_provisioning} -eq 1 ]];then
+% ohpc_indent 5
+% ohpc_validation_comment Add GRUB2 bootloader and update node object parameters
 \begin{lstlisting}[language=bash,literate={-}{-}1,keywords={},upquote=true]
 # Add GRUB2 bootloader and re-assemble VNFS image
 [sms](*\#*) (*\chrootinstall*) grub2
 [sms](*\#*) wwvnfs  -y --chroot $CHROOT
-\end{lstlisting}
-
-\noindent Enabling stateful nodes also requires additional site-specific, disk-related
-parameters in the Warewulf configuration. In the example that follows, the
-compute node configuration is updated to define where to install the GRUB
-bootloader, which disk to partition, which partition to format, and what the
-filesystem layout will look like.
 
-\begin{lstlisting}[language=bash,literate={-}{-}1,keywords={},upquote=true]
 # Update node object parameters
 [sms](*\#*) export sda1="mountpoint=/boot:dev=sda1:type=ext3:size=500"
 [sms](*\#*) export sda2="dev=sda2:type=swap:size=32768"
 [sms](*\#*) export sda3="mountpoint=/:dev=sda3:type=ext3:size=fill"
-[sms](*\#*) wwsh -y object modify -s bootloader=sda -t node "${compute_regex}" 
-[sms](*\#*) wwsh -y object modify -s diskpartition=sda -t node "${compute_regex}" 
-[sms](*\#*) wwsh -y object modify -s diskformat=sda1,sda2,sda3 -t node "${compute_regex}" 
-[sms](*\#*) wwsh -y object modify -s filesystems="$sda1,$sda2,$sda3" -t node "${compute_regex}" 
+[sms](*\#*) wwsh -y object modify -s bootloader=sda -t node "${compute_regex}[1-${num_computes}]" 
```

Comment:
I think the intent was for ${compute_regex} to be a regex that maps to all computes. So, do we really need/want to replace ${compute_regex} with ${compute_regex}[1-${num_computes}] here?